mapie/multi_label_classification.py

```diff
@@ -138,12 +146,16 @@ def __init__(
         estimator: Optional[ClassifierMixin] = None,
         n_jobs: Optional[int] = None,
         random_state: Optional[Union[int, np.random.RandomState]] = None,
-        verbose: int = 0
+        verbose: int = 0,
+        metric_control: Optional[str] = 'recall',
+        method: Optional[str] = None
```

Comment:
UPD: they should be defined just after `estimator` (l146) and before `n_jobs` (l147).

Reply:
Done